src/ztc/api/serializers/roltype.py

```diff
@@ -47,3 +48,13 @@ class Meta:
                 'lookup_field': 'uuid'
             }
         }
+
+    @transaction.atomic()
+    def create(self, validated_data):
```

Comment:
ZRC has the library `drf-writable-nested` to facilitate this via a mixin :smiley:

Reply:
replace manual `create`with `NestedCreateMixin` from drf-writable-nested for `RolType` and `ZaakType` serializers